toxcore/group_common.h

```diff
@@ -32,7 +32,7 @@
 #define MAX_GC_MESSAGE_SIZE GROUP_MAX_MESSAGE_LENGTH
 #define MAX_GC_MESSAGE_RAW_SIZE (MAX_GC_MESSAGE_SIZE + GC_MESSAGE_PSEUDO_ID_SIZE)
 #define MAX_GC_CUSTOM_LOSSLESS_PACKET_SIZE 1373
-#define MAX_GC_CUSTOM_LOSSY_PACKET_SIZE MAX_GC_PACKET_CHUNK_SIZE
+#define MAX_GC_CUSTOM_LOSSY_PACKET_SIZE 1000
```

Comment:
Is there a reason why it can't be (at least) the same as lossless packet size?

Reply:
there is no reason. In theory it could be even larger, if the network supports jumbo frames.

Reply:
It can't be much larger, because it won't fit into the next layer of tox packets. There are reasons for these numbers. Ideally we'd compute the numbers from a single MTU we set for tox packets, and everything else is derived from that.

Reply:
yea.

should i define `MAX_GC_CUSTOM_LOSSY_PACKET_SIZE` in terms of `MAX_GC_CUSTOM_LOSSLESS_PACKET_SIZE` or by the number directly?